bst.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var recursion = function(root, min, max) {
+    if (!root) return true;
+    return root.val < max && root.val > min
+        ? recursion(root.left, min, root.val) && recursion(root.right, root.val, max)
+        : false;
+};
+
+var isValidBST = function(root) {
```

Comment:
Some logic here and in the recursion is duplicated os it's possible to rewrite it like that:
```suggestion
var isValidBST = root => recursion(root, -Infinity, Infinity);
```